### DISCUSSION TITLE
how does one style code blocks?

### DISCUSSION THREAD

**Discussion by LonelyMusicDisc** [TriliumNext/Trilium#2438] [General]
![image](https://user-images.githubusercontent.com/37805980/145876197-beec4a12-8f4c-44d7-8c91-967987b0c05f.png)
so I'm editing a theme and realized that the code blocks are... not fitting, but I don't know the css code to change their background color, any help?

**Comment by Lolabird** [comment]
There may be a better way to edit it, but I used

- `.ck.ck-editor__editable pre {}` for the block itself (sans text),
- `.ck-content pre code  {}` for the text in the block, and 
- `pre::after{}` for the code block label.

**Reply by Lolabird** [reply]
Just realized you asked for background color specifically. You can use `.ck.ck-editor__editable pre {
    background: insert-color-here;}`.
